packages/core/scripts/local/DeployEMP.js

```diff
@@ -115,16 +111,16 @@ const deployEMP = async callback => {
 
     let _emp = await expiringMultiPartyCreator.createExpiringMultiParty.call(constructorParams, { from: deployer });
     await expiringMultiPartyCreator.createExpiringMultiParty(constructorParams, { from: deployer });
-    emp = await ExpiringMultiParty.at(_emp);
+    const emp = await ExpiringMultiParty.at(_emp);
 
     const empConstructorParams = {
       ...constructorParams,
-      finderAddress: Finder.address,
-      tokenFactoryAddress: TokenFactory.address,
+      finderAddress: finder.address,
+      tokenFactoryAddress: tokenFactory.address,
```

Comment:
Note: when we initialize using `getTruffleContract`, we have to use `Finder.deployed()` rather than `Finder.address` because truffle needs an async operation to detect the network.